services/core/ActuatorAgent/tests/test_actuator_pubsub.py

```diff
@@ -146,7 +146,6 @@ def publish_agent(request, volttron_instance):
     # Reset platform driver config store
     cmd = ['volttron-ctl', 'config', 'delete', PLATFORM_DRIVER, '--all']
     process = Popen(cmd, env=volttron_instance.env,
-                    cwd='scripts/scalability-testing',
```

Comment:
`cwd` is not needed here because reseting the driver config store does not require a path to a file.